### ISSUE TITLE
Missing focus ring for stanford only icon

### ISSUE THREAD

**Issue by dnoneill** [sul-dlss/SearchWorks#4977]
https://github.com/user-attachments/assets/178e1383-629a-468a-92bf-b484ea513cf1
